### DISCUSSION TITLE
renew acadmic affiliation problem

### DISCUSSION THREAD

**Discussion by Lossita** [community/community#54733] [GitHub Education]
### Hi everyone,

today i got an email titled "Coupon expiration reminder". i want to use my student pack to continue Github Pro but i found that i cannot reverify my acadmic affilication. in https://education.github.com/discount_requests/application page, after i input my content i found that i cannot press continue button and i was told " You were last verified as a student on Sep 27, 2022. It is not necessary for you to reverify at this time. There may be a wait period between verification and access to academic benefits". and when i want to use my github student pack to activate Temius student pack i was told my student pack is inavaliable. but i cannot update my student pack as above. so how could i do?

and i remembered that two weeks ago i got a mail said you asked me to update the acadmic verification and when i logged to the site i got the same "You were last verified as a student on Sep 27, 2022. It is not necessary for you to reverify at this time. There may be a wait period between verification and access to academic benefits"

**Comment by Lossita** [comment]
now it doesnt show me the date even. Someone can help me?
![QQ截图20230509150455](https://user-images.githubusercontent.com/70831998/237020291-10074322-3c7e-4645-b43e-8278d0ddcf7a.png)

**Comment by samus-aran** [comment]
Hi @Lossita 

Can you be more explicit on the email you received. "Coupon expiration reminder" refers to what? Copilot perhaps? If you are still receiving the notification on the application page that you are still verified you are still receiving your benefits.
